website/sidebars.js

```diff
@@ -321,34 +320,36 @@ module.exports = {
 					type: 'category',
 					label: 'Globals',
 					items: [
-						'_ACTION',
-						'_ARGS',
-						'_MAIN_SCRIPT_DIR',
-						'_MAIN_SCRIPT',
-						'_OPTIONS',
-						'_OS',
-						'_PREMAKE_COMMAND',
-						'_PREMAKE_DIR',
-						'_PREMAKE_VERSION',
-						'_TARGET_ARCH',
-						'_TARGET_OS',
-						'iif',
-						'include',
-						'includeexternal',
-						'printf',
-						'require',
-						'verbosef',
+						'globals/_ACTION',
+						'globals/_ARGS',
+						'globals/_MAIN_SCRIPT_DIR',
+						'globals/_MAIN_SCRIPT',
+						'globals/_OPTIONS',
+						'globals/_OS',
+						'globals/_PREMAKE_COMMAND',
+						'globals/_PREMAKE_DIR',
+						'globals/_PREMAKE_VERSION',
+						'globals/_TARGET_ARCH',
+						'globals/_TARGET_OS',
+						'globals/_WORKING_DIR',
+						'globals/dofileopt',
+						'globals/iif',
+						'globals/include',
+						'globals/includeexternal',
+						'globals/printf',
+						'globals/require',
+						'globals/verbosef',
```

Comment:
I believe you can replace the list of items with a single item:
```js
{
  type: 'autogenerated',
  dirName: 'globals',
},
```
And it should expand to all of the files in the folder. See [here](https://docusaurus.io/docs/sidebar/autogenerated) under the "A real-world example" collapsed section.

@KyrietS I recall when you first introduced this doc system that we had discussed autogenerating the sidebar, was there a reason we didn't do this before? I vaguely recall that it might not have been available at the time, is that right?